.github/scripts/auto-backport.py

```diff
@@ -150,7 +150,10 @@ def main():
         start_commit, end_commit = args.commits.split('..')
         commits = repo.compare(start_commit, end_commit).commits
         for commit in commits:
-            for pr in commit.get_pulls():
+            match = re.search(rf"Closes .*#([0-9]+)", commit.commit.message, re.IGNORECASE)
```

Comment:
What about the `Closes: #` and I think maybe also `Closes scylladb/scylla-enterprise#` like here: https://github.com/scylladb/scylladb/pull/20443

Reply:
> What about the `Closes: #` and I think maybe also `Closes scylladb/scylla-enterprise#` like here: #20443

This is for the closes syntax we get from `pull_github_pr.sh` script, I will check what happened if it's duplicated (although we shouldn't use closes)